webpack.config.js

```diff
@@ -20,7 +20,13 @@ var config = {
     extensions: [".webpack.js", ".web.js", ".ts", ".tsx", ".js"],
     modules: [
       "node_modules",
-      "src"]
+      "src",
+    ]
+  },
+  node: {
+    fs: "empty",
```

Comment:
We can't use the same code in the api and detect the current environment
browser -> use window....
nodejs -> ws
In this case, we could use the same js distribution no ?